packages/config/config/devices/0x0258/motion_sensor_2.json

```
@@ -0,0 +1,158 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "Motion Sensor 2",
+	"description": "PIR Motion Sensor 2",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x008d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity Level Setting",
+			"description": "Defines the sensitivity of PIR sensor.",
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 12,
+			"unsigned": true
+		},
+		"2": {
+			"label": "ON/OFF Duration Setting",
+			"description": "Determines how long the associated devices should stay ON status",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 30
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"description": "Basic Set Command will be sent when motion is triggered",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		"4": {
+			"label": "Enable/Disable PIR Function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable PIR Detector Function",
+					"value": 0
+				},
+				{
+					"label": "Enable PIR Detector Function",
+					"value": 255
+				}
+			]
+		},
+		"5": {
+			"label": "Lux Level Set",
+			"description": "Set a lux level value which determines when the light sensor is activated.",
```

```suggestion
			"label": "Light Sensor Brightness Threshold",
			"description": "The light sensor is active when the brightness is above this level",
```
Not sure about the description - is that correct or the other way round?